documentation/bolt_connect_puppetdb.md

```diff
@@ -91,6 +94,133 @@ puppetdb:
   token: ~/.puppetlabs/token
 ```
 
+## Configuring multiple PuppetDB instances
+
+The Bolt PuppetDB Client supports connections to multiple PuppetDB instances. To
+configure multiple PuppetDB instances, add the `puppetdb-instances` section to
+your [Bolt config](configuring_bolt.md).
+
+The `puppetdb-instances` section is a map of configuration, where each key is the
+name of the PuppetDB instance and values are the configuration for the instance.
+Each instance supports the same configuration as the `puppetdb` section.
+
+For example, to configure a named instance that uses certificate authentication
+and a second instance that uses PE RBAC authentication:
+
+```yaml
+puppetdb-instances:
+  instance-1:
+    server_urls: ["https://instance-1.example.com:8081"]
+    cacert: /etc/puppetlabs/puppet/ssl/certs/ca.pem
+    cert: /etc/puppetlabs/puppet/ssl/certs/my-host.example.com.pem
+    key: /etc/puppetlabs/puppet/ssl/private_keys/my-host.example.com.pem
+  instance-2:
+    server_urls: ["https://instance-2.example.com:8081"]
+    cacert: /etc/puppetlabs/puppet/ssl/certs/ca.pem
+    token: ~/.puppetlabs/token
+```
+
+## Connecting to a named PuppetDB instance
+
+When using Bolt features that connect to PuppetDB, you can specify a named
+instance to connect to if you have configured multiple PuppetDB instances
+under the `puppetdb-instances` section.
+
+To specify a PuppetDB instance to the `puppetdb_*` plan functions, pass the
+PuppetDB instance name as the last positional argument to the function:
+
+```puppet
+plan example() {
+  puppetdb_fact(['host.example.com'], 'instance-1')
+}
+```
+
+To specify a PuppetDB instance to the `apply` plan function, use the `_pdb_instance`
```

Comment:
Should this just be `_puppetdb` to match the cli option?

Reply:
Updated. Also added some documentation I missed to the plan functions page.